Kernel/Devices/PATAChannel.cpp

```diff
@@ -116,11 +116,14 @@ void PATAChannel::initialize(bool force_pio)
             kprintf("PATAChannel: PATA Controller found! id=%w:%w\n", id.vendor_id, id.device_id);
         }
     });
+
+    m_prdt_page = MM.allocate_supervisor_physical_page();
+    m_prdt = reinterpret_cast<PhysicalRegionDescriptor*>(m_prdt_page.ptr()->paddr().get());
```

Comment:
Instead of putting this in a dedicated member variable, let's just have a getter function:

```cpp
PhysicalRegionDescriptor& prdt() { return *reinterpret_cast<PhysicalRegionDescriptor*>(m_prdt_page->paddr().as_ptr()); }
```